src/components/InputPassword/InputPassword.jsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Constants from '../../constants';
+import './style.css';
+
+const InputPassword = ({ title }) => {
+
+  const [showPassword, setShowPassword] = React.useState(false);
+
+  const validate = () => {
+
+  };
+
+  const handle = () => {
+    if (showPassword) {
+      document.getElementById('login-password').type = 'text';
```

Comment:
ref?

Reply:
Yes, it's a good note. I read about Refs and DOM from ReactJS docs. I will fix it in the new PR. Thanks.